### PR TITLE
Add missing concepts

### PR DESCRIPTION
**Pull request by KiithNabaal**
#183

### CODE REVIEW

File: bricksrc/properties.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from rdflib import Literal
-from .namespaces import A, OWL, RDFS, BRICK, VCARD, UNIT
+from .namespaces import A, OWL, RDFS, BRICK, VCARD, UNIT, TAG
```
Comment (gtfierro) [inline bricksrc/properties.py:2]: Does `TAG` get used in `properties.py`? If not, it should be removed

File: generate_brick.py [hunk 2]
```diff
@@ -407,7 +407,7 @@ def add_definitions():
     for o in filter(lambda x: x != pc, pointclasses):
         G.add((BRICK[pc], OWL.disjointWith, BRICK[o]))
 
-logging.info("Defining Equipment, System and Location subclasses")
+logging.info("Defining Equipment, Controller, System and Location subclasses")
```
Comment (gtfierro) [inline generate_brick.py:410]: Let's remove the "Controller" piece from this comment for now given that we aren't adding controllers (yet)

File: tests/test_class_structure.py [hunk 3]
```diff
@@ -50,7 +50,7 @@ def test_subclasses():
     sc1 = [x[0] for x in subclasses1]
     sc2 = [x[0] for x in subclasses2]
     diff = set(sc1).difference(set(sc2))
-
+    print(str(diff))
```
Comment (gtfierro) [inline tests/test_class_structure.py:53]: Can we remove this print statement now that debugging is done?

**Review by gtfierro — CHANGES_REQUESTED** [review]
Really nice job @KiithNabaal !

A few comments/changes:

- can you remove your changes to the `alignments/rec` files? I'd like to keep the changes isolated to the concepts targeted by the PR
- need textual definitions:
    - [x] `Building_Air_Humidity_Setpoint`
    - [x] `Bypass_Air_Humidity_Setpoint`
    - [x] `Discharge_Air_Humidity_Setpoint`
    - [x] `Electric_Baseboard_Radiator`
    - [x] `Exhaust_Air_Humidity_Setpoint`
    - [x] `Hot_Water_Baseboard_Radiator`
    - [x] `Mixed_Air_Humidity_Setpoint`
    - [x] `Outside_Air_Humidity_Setpoint`
    - [x] `Return_Air_Humidity_Setpoint`
    - [x] `Steam_Baseboard_Radiator`
    - [x] `Supply_Air_Humidity_Setpoint`
    - [x] `Supply_Air_Temperature_Setpoint`
    - [x] `Supply_Chilled_Water_Temperature_Setpoint`
    - [x] `Supply_Hot_Water_Temperature_Setpoint`
    - [x] `Zone_Air_Humidity_Setpoint`

- radiators:
    - probably makes sense to include an `Electric_Radiator`
    - make sure to point the subclasses of `Baseboard_Radiator` to their parents:
        ```python
        "Steam_Baseboard_Radiator": {
            "tags": [ ... etc ... ],
            "parents": [BRICK.Steam_Radiator],
        }
        ```

File: bricksrc/properties.py (hunk 1 above)
Comment (KiithNabaal) [reply bricksrc/properties.py:2]: Oh, that was leftover from what I was doing the Controller stuff. Will remove it.

File: tests/test_class_structure.py (hunk 3 above)
Comment (KiithNabaal) [reply tests/test_class_structure.py:53]: Yep!

**Comment by KiithNabaal** [comment]
Hey Gabe,

So I made all requested changes except the ones in the alignments/rec rdf files. I didn't remove the changes because when I was looking over what was changed in git, and I don't recall ever making those changes myself. Could those have been put in from a script at some point when I ran make or make test?

**Comment by gtfierro** [comment]
My guess is your editor automatically applied some style/linting updates to those files

**Comment by KiithNabaal** [comment]
Looks like it was just tabs and an extra line that was added.

**Review by gtfierro — APPROVED** [review]

**Review by gtfierro — APPROVED** [review]
Great job @KiithNabaal !

**Pull request merged**